Bookish/Bookish.DataAccess/BarcodeService.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Drawing.Imaging;
+using System.IO;
+using System.Linq;
+using System.Text;
+using BarcodeLib;
+
+namespace Bookish.DataAccess
+{
+    public interface IBarcodeService
+    {
+        IEnumerable<int> GetBarcodeIds(string isbn, int copies, string imageFolder);
+    }
+    
+    public class BarcodeService : IBarcodeService
+    {
+        private readonly ILibraryService libraryService;
+        public BarcodeService(ILibraryService libraryService)
+        {
+            this.libraryService = libraryService;
+        }
+
+        public IEnumerable<int> GetBarcodeIds(string isbn, int copies, string imageFolder)
```

Comment:
I would expect this to be called something like `GetNewBooksWithBarcodes`. Because you probably want to return some metadata to display as well, e.g. the book title. And why does this take a copies argument? Don't you just want to get all copies? 🙂

Reply:
I initially had it this way as it generates barcodes only for the new books (i.e. if you add more copies of an existing book) but i realise now that that functionality isn't actually implemented. Would you recommend I just remove this and also change the GetNewIds result to just return all of the copy id's rather than the new ones

Reply:
Yes, I would 🙂